android/app/src/main/res/values/donottranslate.xml

```diff
@@ -63,4 +63,6 @@
   <string name="auto_enum_value" translatable="false">AUTO</string>
   <string name="placepage_behavior" translatable="false">com.google.android.material.bottomsheet.BottomSheetBehavior</string>
   <string name="car_notification_channel_name" translatable="false">CAR_NOTIFICATION_CHANNEL</string>
+
+  <string name="android_auto_fgs_explanation_for_special_use" translatable="false">Since Android 14, every Foreground Service must have a type. There is no specific service type for Android Auto.</string>
```

Comment:
Let's also add a link to the original issue: https://issuetracker.google.com/issues/322622698